static/js/activity.js

```diff
@@ -256,7 +256,7 @@ function actually_update_users(user_list) {
         // Render right panel partially
         $.each(user_info, function (index, user) {
             var user_index = all_users.indexOf(user.email);
-            $('#user_presences').find('[data-email=' + user.email + ']').remove();
+            $('#user_presences').find('[data-email="' + user.email + '"]').remove();
```

Comment:
Hmm, interesting; I was getting tracebacks when I tried deploying the old version and this fixed them, so there must be a case (maybe with some browsers) where it doesn't work?